packages/fxa-auth-server/bin/key_server.js

```diff
@@ -110,6 +114,10 @@ async function run(config) {
       };
   Container.set(AccountEventsManager, accountEventsManager);
 
+  const kyselyDb = await setupAccountDatabase(config.database.mysql.auth);
+  const backupCodeManager = new BackupCodeManager(kyselyDb);
```

Comment:
nit: Maybe give this a better name? 

I also think something else might have landed on main that is already calling setupAccountDatabase. Double check before landing.